Resources calendar link creation and removal works

pnuz3n opened the pull request:
This will create button that requests Respa to perform linking between resource and outlook calendar.

![create-linking](https://user-images.githubusercontent.com/642540/98906794-69146180-24c6-11eb-9be1-4f03b6c0d3ee.png)

![remove linking](https://user-images.githubusercontent.com/642540/98906801-6c0f5200-24c6-11eb-8c14-940b23bac875.png)

pnuz3n committed:
Resource can be linked to Outlook calendar

Resource page contains button that can be used to create (or remove)
link between resource and user's Outlook calendar.

pnuz3n commented:
This will create button that requests Respa to perform linking between resource and outlook calendar.

![create-linking](https://user-images.githubusercontent.com/642540/98906794-69146180-24c6-11eb-9be1-4f03b6c0d3ee.png)

![remove linking](https://user-images.githubusercontent.com/642540/98906801-6c0f5200-24c6-11eb-8c14-940b23bac875.png)

SanttuA commented on app/resource-outlook-linker/actions.js:
```diff
@@ -0,0 +1,108 @@
+    throw new TypeError(`Expected string for resourceId but got something else. (Value was ${resourceId}).`);
+  }
+  if (!isInteger(linkId)) {
+    throw new TypeError(`Expected string for resourceId but got something else. (Value was ${linkId}).`);
```
Small typos in error message. We are expecting linkId integer.

pnuz3n replied:
> Unit tests need to be updated and pass. Tests can be run with command `npm test`.

Should be fix'd now.

SanttuA commented:
Unit tests need to be updated and pass. Tests can be run with command `npm test`.

SanttuA commented on app/pages/resource/resource-header/ResourceHeader.js:
```diff
@@ -47,6 +51,21 @@ function ResourceHeader({
 
+  let linkButton;
+  if (outlookLinkExists) {
+    linkButton = (<Button
```
Linter errors prevent build

pnuz3n replied:
Ups.. Will fix asap.

pnuz3n replied:
Had some unforeseen delays during the day, but fixed all lint problems cause by my code.

pnuz3n committed:
Resource's calendar link button fits to the layout

Previously chaning screen width would cause buttons to be rendered
akwardly. This has now been fixed.
